src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1369,15 +1369,8 @@ julia> unique!(df)  # modifies df
 """
     hcat(df::AbstractDataFrame...;
          makeunique::Bool=false, copycols::Bool=true)
-    hcat(df::AbstractDataFrame..., vs::AbstractVector;
-         makeunique::Bool=false, copycols::Bool=true)
-    hcat(vs::AbstractVector, df::AbstractDataFrame;
-         makeunique::Bool=false, copycols::Bool=true)
-
-Horizontally concatenate `AbstractDataFrames` and optionally `AbstractVector`s.
 
-If `AbstractVector` is passed then a column name for it is automatically generated
-as `:x1` by default.
+Horizontally concatenate `AbstractDataFrames`.
```

Comment:
While we're at it, maybe better use user-friendly phrasing, the type is shown in the signature anyway.
```suggestion
Horizontally concatenate data frames.
```